src/tests/unit/data/spot-schema.json

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
Please add a .txt file adjacent to this doc so we know where this file came from, with a link back to the SPOT repo.

Reply:
added